docs/build/start-building/interacting-with-the-network.md

```diff
@@ -26,23 +26,40 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
-## Devnet
+## Devnets
 
-In order to make testing easier, a live devnet (developer network) is now available for use. We call this network the Nerpa Devnet (the 'nerpa' is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
+#### Nerpa Devnet
```

Comment:
Could these be moved up to `###` level? They feel a bit on the small side to me, think they may get lost in the surrounding text